macOS/DuckDuckGo/CrashReports/Model/CrashReportReader.swift

```diff
@@ -17,68 +17,97 @@
 //
 
 import Foundation
+import Common
 
 final class CrashReportReader {
 
-    static let displayName = Bundle.main.displayName
-    static let vpnExtensionDisplayName = "com.duckduckgo.macos.vpn.network-extension"
+    static func validBundleIdentifiers() -> [String] {
+        return [
+            Bundle.main.bundleIdentifier,
+            Bundle.main.vpnMenuAgentBundleId,
+            Bundle.main.vpnSystemExtensionBundleId,
+            Bundle.main.vpnProxyExtensionBundleId,
+            Bundle.main.dbpBackgroundAgentBundleId
```

Comment:
This can get out of sync over time, open to a better approach on how to handle it.

Reply:
Great catch, let me fix that. Thanks!